src/components/unsigned-sweep.js

```diff
@@ -192,6 +260,17 @@ class UnsignedSweep extends Component {
           />
           }
 
+          {this.requiredParams[this.state.coin].includes('userKeyID') &&
```

Comment:
It's a bit confusing that the User key id is optional, but it's part of the required params.